src/tool_source/github.rs

```diff
@@ -18,20 +19,24 @@ const APP_NAME: &str = "LPGhatguy/aftman";
 
 pub struct GitHubSource {
     client: Client,
+    token: Option<String>,
 }
 
 impl GitHubSource {
-    pub fn new() -> Self {
+    pub fn new(auth: Option<AuthManifest>) -> Self {
```

Comment:
The github source probably doesn't need to know about the entire auth manifest, just its own personal access token

```suggestion
    pub fn new(auth: Option<String>) -> Self {
```